Formula/zeromq.rb

```diff
@@ -11,47 +12,56 @@ class Zeromq < Formula
     sha256 "78c24719c7c8dbba157ca225b41d515190281eb27549caaa60a359ec9f33e3f1" => :yosemite
   end
 
-  head do
-    url "https://github.com/zeromq/libzmq.git"
-
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-  end
-
   option :universal
   option "with-libpgm", "Build with PGM extension"
   option "with-norm", "Build with NORM extension"
+  option "with-drafts", "Build and install draft classes and methods"
```

Comment:
```
/******************************************************************************/
/*  These functions are DRAFT and disabled in stable releases, and subject to */
/*  change at ANY time until declared stable.                                 */
/******************************************************************************/```

https://github.com/zeromq/libzmq/blob/06aeb47d73de5ee07de861e9f8dc3053154b5146/src/zmq_draft.h#L33-L36

The configure script enables it if you're building from a git repository or if you request the option.